themes/default/main.tpl

```diff
@@ -11,7 +11,7 @@
 <meta name="referrer" content="origin" />
 <meta name="referrer" content="same-origin" />
 <meta name="generator" content="my little forum {$settings.version}" />
-<link rel="stylesheet" type="text/css" href="{$FORUM_ADDRESS}/{$THEMES_DIR}/{$theme}/style.min.css" media="all" />
+<link rel="stylesheet" type="text/css" href="{$FORUM_ADDRESS}/{$THEMES_DIR}/{$theme}/style.css" media="all" />
```

Comment:
Please add the changes from style.css also to style.min.css and remove this change.